torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -1137,65 +1137,154 @@ RegisterOperators reg2({
         aliasAnalysisFromSchema()),
 });
 
-bool simpleClassTypeArg(const Argument& arg, const ClassTypePtr& type) {
-  return arg.type() == type && !arg.kwarg_only() && !arg.default_value();
+bool isSortableTupleType(
```

Comment:
It's surprising to me that we have a separate `isSortableTupleType` - could we refactor this to `isSortableType`, and within the Tuple Case recurse on the type members?

Reply:
done.